sale_order_action_invoice_create_hook/hooks.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import _
+from odoo.exceptions import UserError
+from odoo.tools import float_is_zero
+from odoo.addons.sale.models.sale import SaleOrder
+
+
+def post_load_hook():
+
+    def new_action_invoice_create(self, grouped=False, final=False):
```

Comment:
Thanks for making this a post_load monkeypatch! To prevent problems with multiple databases on the same instance, please store the original method on the SaleOrder object as well and call that if `self` does not have _get_invoice_group_key (which will be the case on databases that don't have this module installed):
```
if not hasattr(SaleOrder, 'action_invoice_create_original', False):
    SaleOrder.action_invoice_create_original = SaleOrder.action_invoice_create
```